src/server/chromium/videoRecorder.ts

```diff
@@ -63,31 +63,41 @@ export class VideoRecorder {
     //   $ ./third_party/ffmpeg/ffmpeg-mac -h encoder=vp8
     // 3. A bit more about passing vp8 options to ffmpeg.
     //   https://trac.ffmpeg.org/wiki/Encode/VP8
+    // 4. Tuning for VP9:
```

Comment:
We use vp8 not vp9, don't we? Maybe it's time to switch to vp9?

Reply:
That's on the agenda.